server/graphql/v2/query/index.js

```diff
@@ -39,6 +40,7 @@ const query = {
   orders: OrdersCollectionQuery,
   organization: OrganizationQuery,
   project: ProjectQuery,
+  tagFrequency: TagFrequencyQuery,
```

Comment:
- We're usually naming collections `sCollectionQuery`, so here that would be TagFrequenciesCollectionQuery
- But why name it `tagFrequency` if it's a collection of `TagStats`, tagStats would make more sense, so `tagStats: TagStatsCollectionQuery`

Reply:
`tagStats` makes sense; I've refactored it. 👍🏽